<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Member:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_member">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Base QLineEdit styling */
QLineEdit {
    background-color: #fff;           /* white background */
    border: 1px solid #ccc;           /* light gray border */
    border-radius: 6px;               /* rounded corners */
    padding: 6px 8px;                 /* vertical/horizontal padding */
    font-size: 14px;                  /* sensible text size */
    color: #333;                      /* dark text */
}

/* Hover state: slightly darker border */
QLineEdit:hover {
    border: 1px solid #999;
}

/* Focus state: blue glow and stronger border */
QLineEdit:focus {
    border: 1px solid #409EFF;        /* blue accent */
    outline: none;                    /* remove default focus ring */

}

/* Disabled state: muted look */
QLineEdit:disabled {
    background-color: #f5f5f5;
    border: 1px solid #e0e0e0;
    color: #999;
}

/* Placeholder text styling */
QLineEdit[echoMode=&quot;0&quot;] {            /* only applies when not password mode */
    qproperty-placeholderText: &quot;Enter text…&quot;;
}
QLineEdit::placeholder {
    color: #999;
    font-style: italic;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter text…</string>
   </property>
  </widget>
  <widget class="QPushButton" name="member_btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>200</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:gray;
border: none;
color: white;</string>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_relationship">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Base QLineEdit styling */
QLineEdit {
    background-color: #fff;           /* white background */
    border: 1px solid #ccc;           /* light gray border */
    border-radius: 6px;               /* rounded corners */
    padding: 6px 8px;                 /* vertical/horizontal padding */
    font-size: 14px;                  /* sensible text size */
    color: #333;                      /* dark text */
}

/* Hover state: slightly darker border */
QLineEdit:hover {
    border: 1px solid #999;
}

/* Focus state: blue glow and stronger border */
QLineEdit:focus {
    border: 1px solid #409EFF;        /* blue accent */
    outline: none;                    /* remove default focus ring */

}

/* Disabled state: muted look */
QLineEdit:disabled {
    background-color: #f5f5f5;
    border: 1px solid #e0e0e0;
    color: #999;
}

/* Placeholder text styling */
QLineEdit[echoMode=&quot;0&quot;] {            /* only applies when not password mode */
    qproperty-placeholderText: &quot;Enter text…&quot;;
}
QLineEdit::placeholder {
    color: #999;
    font-style: italic;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter text…</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Relationship:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_btn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#0B2E33;
color: white</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #1ABC9C	;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none; color: white;</string>
      </property>
      <property name="text">
       <string>REGISTER HOUSEHOLD MEMBER</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
